### ISSUE TITLE
Error while instaling

### ISSUE THREAD

**Issue by stereobooster** [TwP/inifile#10]
```
$ bundle install
Invalid gemspec in [/var/lib/gems/1.8/specifications/inifile-1.1.0.gemspec]: invalid date format in specification: "2012-02-28 00:00:00.000000000Z"
Invalid gemspec in [/var/lib/gems/1.8/specifications/inifile-1.1.0.gemspec]: invalid date format in specification: "2012-02-28 00:00:00.000000000Z"
Invalid gemspec in [/var/lib/gems/1.8/specifications/inifile-1.1.0.gemspec]: invalid date format in specification: "2012-02-28 00:00:00.000000000Z"
```

additional info

```
$ ruby -v
ruby 1.8.7 (2011-06-30 patchlevel 352) [i686-linux]
$ uname -a
3.0.0-15-generic #26-Ubuntu SMP Fri Jan 20 15:59:53 UTC 2012 i686 i686 i386 GNU/Linux
```

**Comment by TwP** [comment]
Which rubygems version are you using?

```
$ gem -v
1.8.24
```

I cannot find that date string anywhere in the gemspec that is shipped with version 1.1.0 of inifile.

**Comment by stereobooster** [comment]
I can't reproduce it anymore because I uninstall 1.8.7 and install 1.9.2 and it works fine. So I will close this issue
